### PR TITLE
ON-2719 Adds discovery-refresh-graph.js

### PR DESCRIPTION
**Pull request by jimturnquist**
(cherry picked from commit 8dd0c7c7716dde6efa12b25f5b4cd4fef04c6710)

### CODE REVIEW

**Comment by benbp** [comment]
:+1:

**Comment by jimturnquist** [comment]
@heckj @benbp I think Joe gave the thumbs up on the 1.1 commit and Ben gave the thumbs up on the master commit.

**Comment by heckj** [comment]
:+1:

File: lib/graphs/discovery-refresh-graph.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Copyright 2015, EMC, Inc.
```
Comment (yyscamper) [inline lib/graphs/discovery-refresh-graph.js]: Copyright 2016

**Comment by yyscamper** [comment]
:+1: after copyright fix.

**Comment by jimturnquist** [comment]
Copyright has been fixed.

**Pull request merged**